mintapi/api.py

```diff
@@ -1,6 +1,7 @@
 import json
 import random
 import time
+import re
```

Comment:
Do you mind making this alphabetical and also removing the pandas import logic and the pandas mention the README?